app/models/energy/schedulebase.py

```diff
@@ -5,28 +5,10 @@
 from uuid import UUID, uuid4
 
 
-class ScheduleContinuous(BaseModel):
-    """This Schedule Numeric Type allows all numbers, including fractional amounts, \
-        within the range to be valid."""
-
-    type: Enum('ScheduleContinuous', {'type': 'ScheduleContinuous'})
-
-    schedule_continuous: float = None
-
-
-class ScheduleDiscrete(BaseModel):
-    """This Schedule Numeric Type allows only integers within the range to be valid."""
-
-    type: Enum('ScheduleDiscrete', {'type': 'ScheduleDiscrete'})
-
-    schedule_discrete: int = None
-
-
-class ScheduleNumericType (BaseModel):
+class ScheduleNumericType (str, Enum):
     """Designates how the range values are validated."""
-
-    numeric_type:  Union[ScheduleContinuous, ScheduleDiscrete]
-
+    schedule_continuous = 'ScheduleContinuous'
+    schedule_discrete = 'ScheduleDiscrete'
```

Comment:
I don't think we need the word `Schedule` here.  Just `Continuous` and `Discrete` should be fine since this is ultimately the only text that makes it into the IDF.